op-node/rollup/async/asyncgossiper_test.go

```diff
@@ -0,0 +1,143 @@
+package async
+
+import (
+	"context"
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/stretchr/testify/require"
+)
+
+type mockNetwork struct {
+	reqs []*eth.ExecutionPayload
+}
+
+func (m *mockNetwork) PublishL2Payload(ctx context.Context, payload *eth.ExecutionPayload) error {
+	m.reqs = append(m.reqs, payload)
+	return nil
+}
+
+// TestAsyncGossiper tests the AsyncGossiper component
+// because the component is small and simple, it is tested as a whole
+// this test starts, runs, clears and stops the AsyncGossiper
+// because the AsyncGossiper is run in an async component, it is tested with eventually
+func TestAsyncGossiper(t *testing.T) {
+	m := &mockNetwork{}
+	// Create a new instance of AsyncGossiper
+	p := NewAsyncGossiper(m)
+	ctx, cancel := context.WithCancel(context.Background())
+
+	// Start the AsyncGossiper
+	p.Start(ctx)
+
+	// Test that the AsyncGossiper is running within a short duration
+	require.Eventually(t, func() bool {
+		return p.running.Load()
+	}, 100*time.Millisecond, 10*time.Millisecond)
```

Comment:
A 100ms duration worries me in CI. If there is no contention, this is a good check, but we run lots of tests in parallel & at once.

Reply:
Yep. I'm saying that it should be like 1s instead of 100ms. We've spent a lot of time increasing timeouts in op-e2e

Reply:
Oh my bad, I thought you wanted to lower this! `1s` it shall be.

Reply:
I'd go with something more like 15s given it will bail out early in the normal case.  CI servers are amazingly slow at times and a 1s timeout can be reached really easily.